### PR TITLE
Add 15min wait message

### LINKED ISSUE #47116: [FE] Day-of Check-in: Add Content for 15-minute Wait Message to Confirmation Page
## Wireframes
[wireframe](https://app.abstract.com/projects/ab30c34e-e2f9-4d3e-bb96-3b683b006c24/branches/9164ff31-83ec-41c5-8841-169077282e59/commits/0db2f1608417d8c6c664f509ee0de6885ca7bc96/files/585436fe-2120-45f4-b4f7-fbae01d258d9/layers/43283FE4-5CEF-4571-9FD0-DDC85FE19503?collectionId=e34f4cad-7268-4a1d-86e7-29e8ec02c1f8&collectionLayerId=f8da92d8-569a-4dd5-a2a9-eeebeeb33370&mode=design)

## Tasks
- [ ] [FE] Add "If you wait more than 15 minutes, tell a staff member." so the end of the Day-of Check-in Confirmation Page message in the blue box for an in-person appointment only
- [ ] Add to (or make sure its in the) [translation doc](https://docs.google.com/spreadsheets/d/1xv63kaIR_UMq1bssSleMODXEV14OOGwfcL41rBBWhwk/edit#gid=14407571)
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

### PR DESCRIPTION
**Pull request by wullaski**
## Description

Adds "If you wait more than 15 minutes, tell a staff member." to the end of the Day-of Check-in Confirmation Page message in the blue box for an in-person appointment only

## Original issue(s)
[https://github.com/department-of-veterans-affairs/va.gov-team/issues/47116](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47116)

## Testing done

Visual

## Screenshots

Screen Shot 2022-10-06 at 11 55 16 AM

## Acceptance criteria
- [ ] Message includes verbiage about 15 minute wait(English only)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js [hunk 1]
```diff
@@ -45,6 +45,12 @@ class Confirmation {
       .and('include.text', 'Go to another appointment');
   };
 
+  validateConfirmationAlert = () => {
+    cy.get('[data-testid="confirmation-alert"]')
+      .invoke('text')
+      .should('have.length.gt', 0);
```
Comment (brianseek) [inline src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js:51]: This seems to work well. Curious about the 0, what is this asserting? Also the `gt`.

Nothing to change, just curious about it. For testing that things exist, we usually just do something like `.should('be.visible')`

**Review by brianseek — CHANGES_REQUESTED** [review]
Looking good, one minor thing on the translation string and a question about the test.

File: src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js (hunk 1 above)
Comment (wullaski) [reply src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js:51]: It's testing that there is text inside the alert. `be.visible` could pass with an empty alert.

Comment (brianseek) [reply src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js:51]: ah that makes sense, thanks for the info. We could probably use this to make some of our other similar tests more robust if we do a testing refactor in the future.

**Review by brianseek — APPROVED** [review]
LGTM!

**Pull request merged**